flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/AbstractUdfStreamOperator.java

```diff
@@ -55,13 +56,18 @@
 	
 	/** the user function */
 	protected final F userFunction;
+
+	protected EventTimeFunction eventTimeFunction;
 	
 	/** Flag to prevent duplicate function.close() calls in close() and dispose() */
 	private transient boolean functionsClosed = false;
 	
 	
 	public AbstractUdfStreamOperator(F userFunction) {
 		this.userFunction = requireNonNull(userFunction);
+		if(userFunction instanceof  EventTimeFunction) {
```

Comment:
Thanks for the super quick review. A quick question - In intellij IDE how should a formatting  be applied?  In Eclipse Ctrl+Shift+F applies the formatter that was configured. How about here?  I can update the next commit based on that. Thank you.